src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -84,6 +85,10 @@ def generate_user_assigned_identity_resource_id(self, resource_group):
             resource_group, identity_name)).get_output_in_json()
         return identity.get("id")
 
+    def _get_test_data_file(self, filename):
```

Comment:
get_test_data_file_path?

file usually means one pointer to one file handler.

Reply:
Good point, changed in commit [386f9d0](https://github.com/Azure/azure-cli/pull/19115/commits/386f9d05646beca7f620c78bab183b97e38a99fb). Changed the function name to `get_test_data_file_path` to make it clearer that this function is only used to get the file path and return a string.